src/roku-types/data.json

```diff
@@ -9595,7 +9654,7 @@
                             "description": "The [roScreen](https://developer.roku.com/docs/references/brightscript/components/roscreen.md or [roImageCanvas](https://developer.roku.com/docs/references/brightscript/components/roimagecanvas.md instance to be associated with this caption renderer.",
                             "isRequired": true,
                             "name": "screen",
-                            "type": "roScreen or roImageCanvas"
+                            "type": "Object"
```

Comment:
"Object" is technically correct, but it would be good to eventually support multiple types for when we get union types in brighterscript. Would it be better to go ahead and store an array of types for this "type" key instead?